Docs-3579 Update Pools Articles (#1350)
truenas/documentation: content/CORE/CORETutorials/Storage/Pools/ManagingPools.md

This option is typically used when virtual disks are resized apart from TrueNAS.
{{< /expand >}}
{{< expand "Upgrade Pool" >}}
This option only displays when the pool can be upgraded to use new [ZFS feature flags]({{< relref "/content/References/ZFSPrimer.md" >}}).
Before upgrading an existing pool, be aware of these caveats:

* Upgrading a pool is one-way. This means that if you change your mind. 
  You cannot go back to an earlier ZFS version or downgrade to an earlier version of the software that does not support those ZFS features.
* Upgrading can affect data. Before performing any operation that can affect the data on a storage disk, always back up all data first and verify the integrity of the backup. 
  While it is unlikely that the pool upgrade affects the data, it is always better to be safe than sorry.
* Upgrading a ZFS pool is optional. Do not upgrade the pool if the possibility of reverting to an earlier version of TrueNAS or repurposing the disks in another operating system that supports ZFS is desired. 
  It is not necessary to upgrade the pool unless the end user has a specific need for the newer ZFS Feature Flags. 

>   If a pool is upgraded to the latest feature flags, it is not possible to import that pool into another operating system that does not yet support those feature flags.

Reviewer: ```suggestion
  If you upgrade a pool to the latest feature flags, you cannot import that pool into another operating system that does not yet support those feature flags.
```
